### PR TITLE
LPS-121997 Update service.xml and add Upgrade processes for Country

### PR DESCRIPTION
**Pull request by alee8888**
@drewbrokke

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:relevant

**Comment by alee8888** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/service.xml [hunk 1]
```diff
@@ -289,21 +289,47 @@
...
+		<!-- Localized entity -->
+
+		<localized-entity>
+			<localized-column name="title" />
+
+			<!-- Finder methods -->
+
+			<finder name="Title" return-type="country">
+				<finder-column name="title" />
+			</finder>
```
Comment (pei-jung) [inline portal-impl/src/com/liferay/portal/service.xml:331]: Is this finder needed? I don't see this in the CommerceCountry service.xml. 

It'd be easier if you leave out all the finders for now and add it later when you work on the service layer, so you know what finders you need.

**Comment by alee8888** [comment]
ci:test:sf

**Comment by alee8888** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
@patriciajeanperez is this a fluke?

**Comment by patriciajeanperez** [comment]
looks like it @pei-jung, will re-run

**Comment by patriciajeanperez** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
@pyoo47 is this in the queue?

**Comment by pyoo47** [comment]
ci:test:relevant

**Comment by pyoo47** [comment]
In the queue here: https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest(master)

We are seeing some sporadic drops on the github webhook lately.

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by patriciajeanperez** [comment]
Fix for the failures have been merged a while ago: https://github.com/brianchandotcom/liferay-portal/pull/95700. I will manually forward this pr since the failures are not related to this pull.

**Comment by pei-jung** [comment]
thanks @patriciajeanperez!

**Comment by patriciajeanperez** [comment]
Pushed to BChan
https://github.com/brianchandotcom/liferay-portal/pull/95705

**Comment by patriciajeanperez** [comment]
ci:close

**Pull request closed without merging**